### DISCUSSION TITLE
Tempo Sizing Reference / Guidance

### DISCUSSION THREAD

**Discussion by vkg23** [grafana/tempo#2836] [General]
Hi , 

We are currently evaluating Tempo for our Observability platform. 
I dont have any Trace count figure at this point , 
Still , Are there any sizing guidelines avaialable for CPU/Mem/Cache/storage that need to be considered for Tempo etc. 
Thank You

**Comment by joe-elliott** [comment]
Unfortunately, we don't have anything at the moment. Tempo performance is improving everyday and it's very difficult to give values that don't immediately become out of date. Additionally, trace write patterns can have a large impact on performance. If you look around you will find many threads with optimization hints and people posting their personal cluster's performance.

If you'd like specific advice I can try to help.

**Comment by electron0zero** [comment]
Hi, we have added a [cluster sizing guide](https://grafana.com/docs/tempo/next/setup/size/) in the tempo docs.

Please use this to size your tempo cluster but please be aware that these are just guidelines and not hard rules, and it might change with new releases.

cc @vkg23 & @troykomodo
